dispatcher.go

```diff
@@ -73,7 +77,7 @@ func (d *Dispatcher) Dispatch(event Event) error {
 	defer d.lock.Unlock()
 	// Key format: <timestamp>/<entity-id>/<type>
 	// @todo: This is up for debate, its a 'fake' Event struct right now anyway
-	key := datastore.NewKey(string(event.Time())).ChildString(event.EntityID()).ChildString(event.Type())
+	key := datastore.NewKey(string(event.Time())).ChildString(event.EntityID().String()).ChildString(event.Type())
```

Comment:
Do you still think having time first is a good idea? I was starting to wonder if prefix searching via entity it would be useful. Then you'd want something like `<entity id>/<time>/<type>/<uid>`?

Reply:
TBH, didn't look much more in the `dispatcher.go` after my last review in your PR.
I'm not sure yet when we're going to query the _raw_ events from the Dispatcher... I guess having the answer to that question would make that decision more clear. 

Do you have a clear situation in mind in which raw events would be queried?

Reply:
Left in #16 to keep a record of this topic.